src/routes/(app)/admin[[hash=admin_hash]]/cms/new/+page.server.ts

```diff
@@ -28,9 +36,14 @@ export const actions = {
 				fullScreen: z.boolean({ coerce: true }),
 				maintenanceDisplay: z.boolean({ coerce: true }),
 				hasMobileContent: z.boolean({ coerce: true }),
-				mobileContent: z.string().max(MAX_CONTENT_LIMIT).optional()
+				mobileContent: z.string().max(MAX_CONTENT_LIMIT).optional(),
+				hasCustomMeta: z.boolean({ coerce: true }),
```

Comment:
You should just ignore this parameter and base everything on `metas.length`.

Same in the update logic.

Currently I think if I remove all the metas of the CMS page in the UI it maybe doesn't work?